### DISCUSSION TITLE
Keep document outline showing in pdf viewer sidebar after rebuilding?

### DISCUSSION THREAD

**Discussion by bagrow** [James-Yu/LaTeX-Workshop#3647] [Q&A] [answered]
When I rebuild my document, the sidebar in the pdf viewer (if it is open) returns to the "Show Thumbnails" view. I find the "Show Document Outline" view more helpful. 

Can the sidebar remember which view it was on after rebuilding? Is there a setting I can change? I couldn't find anything obvious...

I have a vague memory that the sidebar used to stay on the existing view. Even if it lost its place in the outline after rebuilding, it was still more useful to me than seeing the thumbnails. Did an update change this?

**Comment by James-Yu** [comment, marked as answer]
#3649 .
